.github/workflows/bindgen.yml

```diff
@@ -227,6 +227,7 @@ jobs:
           BINDGEN_FEATURE_RUNTIME: ${{matrix.feature_runtime}}
           BINDGEN_FEATURE_EXTRA_ASSERTS: ${{matrix.feature_extra_asserts}}
           BINDGEN_NO_DEFAULT_FEATURES: ${{matrix.no_default_features}}
+          BINDGEN_RUST_FOR_LINUX_TEST: ${{matrix.os == 'ubuntu-latest' && matrix.llvm_version == '16.0' && matrix.feature_extra_asserts == 0 && 1 || 0}}
```

Comment:
Perhaps a single additional job should do this. In any case, by doing it in `test.sh`, it is easy to add it to existing ones or not as needed.

Also, I disable it under `feature_extra_asserts` since it seems to take too long / does not finish.

Reply:
Hi Miguel, do you have any estimates on the time it takes to run this new workflow?

Reply:
If you mean adding it to the existing jobs, then taking a look at the CI run here and comparing it with the latest, it adds about a minute or two.

If you mean moving it to a new job, then it may be able to run in parallel with the existing jobs and the total time of the new job is similar to existing ones (i.e. 2-3 minutes), as long as one disables e.g. `BINDGEN_MAIN_TESTS` and so on.

Reply:
(Edited the message above to add some more details.)

Which one is the best approach I guess depends on how much you care about the extra minute(s) in existing tests and whether you think running this new part under e.g. debug is worth it etc. I am happy to send the "separate job" one where I added a new `BINDGEN_*` variable to skip the other bits if needed.

Reply:
I think we can live with an extra couple minutes here and there, what do you think @emilio? I don't have a strong preference about having this on a completely new job or not. The only question I have is if we want to include this job as a required green check for every PR or if we would be ok with it failing from time to time and fix bindgen before each release.